### DISCUSSION TITLE
Roaming status when UE power off

### DISCUSSION THREAD

**Discussion by Ma-Guang-Han** [camaraproject/DeviceStatus#219] [Q&A]
Dear community, I have a question regarding the Device Roaming Status API. When the UE (mobile device) is powered off, should the roaming status be determined as True or False? Is this determination made by each operator individually, or is there a standard to follow?

**Comment by eric-murray** [comment]
Hi @Ma-Guang-Han

CAMARA APIs are atomic. An application developer is meant to use multiple API calls to fulfil their use case.

If the application developer is concerned as to whether or not the device is switched off, they would use the device reachability API.

As for the roaming status API, there is a `lastStatusTime` field defined for the response, and so the API provider can return the last known status of the device. If the API implementation checks in "real time" and has no historical data, then a device that is switched off is definitely not roaming, because roaming means "connected to a network other than the home network".
